### PR TITLE
support ecto >= 2

### PR DESCRIPTION
**Pull request by amrnt**
As everyone is trying to upgrade to ecto 3, I changed the requirements to be `>= 2.0.0` instead of lock it on ecto2.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -46,7 +46,7 @@ defmodule AbsintheRelay.Mixfile do
   defp deps do
     [
       {:absinthe, "~> 1.5.0-alpha.0"},
-      {:ecto, "~> 2.0", optional: true},
+      {:ecto, ">= 2.0.0", optional: true},
```
Comment (benwilson512) [inline mix.exs]: This range is too large. I have no way of knowing if there will be an Ecto 4 and if Absinthe.Relay will support it. If it works on 3.0 then we can do `"~> 2.0 or ~> 3.0"`

Comment (amrnt) [reply mix.exs]: You are very right. Fixed!

**Comment by benwilson512** [comment]
Thank you!

**Comment by mmrobins** [comment]
Any idea when this might go out in a released version?  I see another commit on master has `{:absinthe, "~> 1.5.0-alpha.0"}`, so maybe this change is caught up in a release cycle for larger changes?

**Comment by bgentry** [comment]
I opened #129 to backport this to the v1.4 release branch.

**Pull request merged**